modular_citadel/code/modules/projectiles/guns/pumpenergy.dm

```diff
@@ -187,7 +193,7 @@
 
 /obj/item/projectile/beam/disabler/slug
 	name = "positron blast"
-	damage = 80
+	damage = 40
```

Comment:
A slug is only one single shot.  Blindly halving the damage comically misses the point of a slug round and completely ignores the fact that the slug costs more to fire. It's supposed to be stronger than the buckshot as it's able to concentrate far more energy into a single round, as splitting the energy results in some overall energy loss.